plugins/dialog/plugin.js

```diff
@@ -176,15 +176,15 @@ CKEDITOR.DIALOG_STATE_BUSY = 2;
 			'</div>';
 
 	function buildDialog( editor ) {
-		var element = CKEDITOR.dom.element.createFromHtml( CKEDITOR.addTemplate( 'dialog', templateSource ).output( {
+		var element = CKEDITOR.dom.element.createFromHtml( CKEDITOR.tools.htmlSafeByReview(CKEDITOR.addTemplate( 'dialog', templateSource ).output( {
 			id: CKEDITOR.tools.getNextNumber(),
 			editorId: editor.id,
 			langDir: editor.lang.dir,
 			langCode: editor.langCode,
 			editorDialogClass: 'cke_editor_' + editor.name.replace( /\./g, '\\.' ) + '_dialog',
 			closeTitle: editor.lang.common.close,
 			hidpi: CKEDITOR.env.hidpi ? 'cke_hidpi' : ''
-		} ) );
```

Comment:
output of a template - legacy

Reply:
We can try to force the template.output to return a TrustedHTML and see what breaks, and unwrap the value if it's used for string manipulation.

Reply:
Legacy converted template.output. Looks like no breakages occur as of now. I'll remove the other template.output justifications unless they involve values that need to be explained or that we've investigated deeply.